backend/services/docker-compose.yml

```diff
@@ -11,10 +11,22 @@ services:
       - APP_NAME=TransactionService
     expose:
       - 5000
-
+  mining:
```

Comment:
I think mining-api would be better in all cases except for the container name

Reply:
I will rename it with my next PR and change the folder location when the blockchain pr gets merged in